### PR TITLE
Prevent auto zoom on iOS browsers

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
On iOS Safari, when a user taps on the text input field in the Country Selection screen, the view zooms in and stays that way for the rest of the flow unless the user adjusts the zoom themselves.

# Solution
Set font size for country selector's text input to be 16px unit.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated? - feature not released yet
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/CountrySelector/style.scss [hunk 1]
```diff
@@ -124,6 +124,8 @@ $_sdk-option-border-bottom: 0;
 .custom__input {
   background-color: transparent;
   position: relative;
+  /* font size for form elements (input, select, textarea) must be at least 16px otherwise iOS Safari intervenes & forces UI to zoom in */
+  font-size: 16 * $unit;
```
Comment (ethan605) [inline src/components/CountrySelector/style.scss:128]: We already have a `$font-size-base` defined [here](https://github.com/onfido/onfido-sdk-ui/blob/dac730f897d15fab17b0e09be2b64eb91bf3a746/src/components/Theme/constants.scss#L13), is it possible to reuse it?
```suggestion
  font-size: $font-size-base;
```

**Review by ethan605 — CHANGES_REQUESTED** [review]

File: src/components/CountrySelector/style.scss (hunk 1 above)
Comment (kopijunkie) [reply src/components/CountrySelector/style.scss:128]: Good catch. I was looking for this but somehow didn't find it.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]
LGTM

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1180-pr-onfido-sdk-ui-onfido.surge.sh

**Review by eraxyz — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1180-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1180-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**